src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/common.hpp

```diff
@@ -103,35 +124,49 @@ static constexpr size_t vec_len_f32_avx2 = vec_len_avx2 / sizeof(float);
         };
         return _mm256_loadu_si256(&mask[N7]);
     }
-    inline __m256 mm256_uni_loadu_ps(const float* a) {
-        return _mm256_loadu_ps(a);
-    }
-    inline void mm256_uni_storeu_ps(float* a,  __m256 v) {
-        _mm256_storeu_ps(a, v);
+
```

Comment:
Are those store kernel just placeholder? If so, please leave an assertion as above.

Reply:
f32, f16 are used in [softmax](https://github.com/luo-cheng2021/openvino/blob/4172152a8e69404722c63c4425eb65a294bc2fe4/src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/softmax_kernel.hpp#L474), but bf16 is placeholder, assert is added.